Source/Widgets/InfoBox/InfoBox.css

```diff
@@ -38,6 +38,10 @@
     box-sizing: content-box;
 }
 
+.cesium-infoBox-title a, .cesium-infoBox-title a:active, .cesium-infoBox-title a:focus {
```

Comment:
I believe it was because the links were very hard to read given the default `a` color, so I made it the color of the rest of the title